docs/packages/dara-components/plotting/introduction.mdx

```diff
@@ -169,10 +169,27 @@ You should use:
 You can choose a default theme for all your plotting components using the following
 
 ```python
-from dara.components.plotting import set_default_bokeh_theme, set_plotly_default_theme
+from dara.components.plotting import set_default_bokeh_theme, set_default_plotly_theme
 
 # provide theme object as expected by bokeh Document, see docs https://docs.bokeh.org/en/2.4.1/docs/reference/document.html#bokeh.document.Document.theme
 set_default_bokeh_theme({ ... })
 
 set_default_plotly_theme({ ... })
 ```
+
+For example, for Plotly, see the following example:
```

Comment:
I think it is still worth adding a little sentence or two here saying what `set_default...` takes, which is just s dict representation of the theme that each plotting library takes. Since they are just dicts people can easily add to them should they wish. 

Some references that could be nice to add:
For Plotly: https://plotly.com/javascript/reference/#layout
For Bokeh: https://docs.bokeh.org/en/latest/docs/reference/themes.html#theme